pages/spicedb/concepts/datastores.mdx

```diff
@@ -30,6 +30,44 @@ spicedb migrate head \
 
 For more information, refer to the [Datastore Migration documentation](/spicedb/concepts/datastore-migrations).
 
+## Read replicas for Postgres and MySQL
+
+SpiceDB can query read-only replicas of permissions data stored in Postgres and MySQL while maintaining consistency guarantees by detecting if a given revision has been replicated from the primary datastore before reading from the replica.
+If the requested revision is not found in the replica, the request will be redirected to the primary datastore.
+
+When using multiple replicas, the queried replica will be chosen in a round-robin fashion.
+Please note that the maximum replica count is 16.
+
+Traffic using `minimize_latency` will benefit the most from using read replicas.
+
+### PostgreSQL read replicas
+
+You can use a load balancer between PostgreSQL replicas and SpiceDB.
+When using a load balancer, please ensure that `ReadStrictMode` is enabled on your datastore.
```

Comment:
That flag does not exist anymore, it's baked into the selected datastore. We don't have to mention it.
```suggestion
```

Reply:
awesome!